modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/QuantitySelector.js

```diff
@@ -68,30 +70,32 @@ function QuantitySelector({
 	// eslint-disable-next-line react-hooks/exhaustive-deps
 	useEffect(willUpdate, [selectedQuantity]);
 
+	const commonProps = {
+		className: classnames({
+			'form-control-lg': large,
+			'quantity-selector': true,
+		}),
+		'data-component-id': componentId,
+		disabled,
+		name,
+		onChange,
+		value: selectedQuantity,
+	};
```

Comment:
This is a really interesting way to handle this, I haven't thought of doing it like this before 👍 👍